### PR TITLE
Added ucb1_tuned player

### PR DESCRIPTION
**Pull request by holdenkold**

### CODE REVIEW

File: player/mcts/mcts_node.py [hunk 1]
```diff
@@ -75,13 +76,15 @@ def build_tree(self, should_continue):
...
                 node = node.parent
 
             iteration += 1
 
     def select_child(self):
-        return max(self.children, key=lambda child: child.uct)
+        return max(self.children, key=lambda child: child.eval)
 
     def add_child(self, action: int, game_state: IGameState):
         child = MctsNode(game_state, action, self)
```
Comment (Michalski-Artur) [inline player/mcts/mcts_node.py:90]: I guess if we want to use other implementations for mcts_node we should also create instance of proper type here as well

File: player/ucb1_tuned/ucb1_tuned_node.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+class Ucb1TunedNode(MctsNode):
+    @property
+    def variance_bound(self):
+        return self.reward_squared / self.number_of_runs - self.sample_mean ** 2
+    
+    
+    @property
+    def eval(self) -> float:
+        ratio_of_visits = np.sqrt(np.log(self.parent.number_of_runs) / self.number_of_runs)
+        return self.sample_mean + np.sqrt(ratio_of_visits * min(0.25, self.action_variance() + 2 * ratio_of_visits))
```
Comment (Michalski-Artur) [inline player/ucb1_tuned/ucb1_tuned_node.py]: What is `self.action_variance()` here? I guess you meant `variance_bound`? So probably `variance_bound` should be renamed to `action_variance`?

**Review by Michalski-Artur — APPROVED** [review]
Except for two things mentioned in comments, looks good 👍

File: player/ucb1_tuned/ucb1_tuned_node.py (hunk 2 above)
Comment (holdenkold) [reply player/ucb1_tuned/ucb1_tuned_node.py]: yes ...

Comment (Michalski-Artur) [reply player/ucb1_tuned/ucb1_tuned_node.py]: Don't you worry :)

**Pull request merged**